torch/_dynamo/variables/torch.py

```diff
@@ -400,9 +400,16 @@ def call_function(
                 tx, args[0], args[1].as_python_constant()
             )
         elif self.value is torch.nn.attention.sdpa_kernel:
-            assert len(args) == 1 or (len(kwargs) == 1 and "backends" in kwargs)
-            backends = args[0] if len(args) == 1 else kwargs["backends"]
-            set_priority = kwargs["set_priority"] if "set_priority" in kwargs else False
+            sig = inspect.signature(torch.nn.attention.sdpa_kernel)
+            try:
+                bound_args = sig.bind(*args, **kwargs)
+                bound_args.apply_defaults()
```

Comment:
Need to be careful here because the defaults are python constants while args/kwargs still use VariableTrackers.

Reply:
args/kwargs are list/dict of VariableTracker. SDPAKernelVariable seems to take Python constants str/bool as keyword arguments. I think the code would be more clear if we avoided apply_defaults and do a manual check to see if arguments["..."] is provided.